<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>424</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>424</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>431</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Task1_Button">
       <property name="text">
        <string>Task 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Task2_Button">
       <property name="text">
        <string>Task 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Task3_Button">
       <property name="text">
        <string>Task 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Task4_Button">
       <property name="text">
        <string>Task 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistic</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Task1">
         <property name="text">
          <string>Task 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Task2">
         <property name="text">
          <string>Task 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Task3">
         <property name="text">
          <string>Task 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Task4">
         <property name="text">
          <string>Task 4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="Label_point1">
           <property name="text">
            <string>task1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Label_point2">
           <property name="text">
            <string>task2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Label_point3">
           <property name="text">
            <string>task3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Label_point4">
           <property name="text">
            <string>task4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Exit_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="Login">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_login">
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBut">
    <property name="title">
     <string>Button</string>
    </property>
    <addaction name="actionTask_1"/>
   </widget>
   <addaction name="menuBut"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTask_1">
   <property name="text">
    <string>Task 1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
